README.md

```diff
@@ -597,7 +597,7 @@ ssh::sshd_config_match:
 
 sshd_config_hostcertificate
 ---------------------------
-Absolute path to the OpenSSH Host CA Certificate (HostCertificate) for use with SSH CA validation for Host Certificates.
+An Array of Absolute Paths to the Host CA Public Key. Each entry *MUST* be tied 1:1 to a Host CA Private Key (see [sshd_config_hostkey](#sshd_config_hostkey))
```

Comment:
Does that matter if the parameter was incorrect in the first place?

I'm fixing it as the original parameter was "broken" already from the SSH standpoint. Meaning the value could only support one HostKey + HostCertificate combination, when in reality each HostCertificate should be directly tied to a HostKey?

So from a Puppet standpoint, this is a breaking change, but from an SSH standpoint this is a bugfix